pkg/cli/init-templates/cog.yaml

```diff
@@ -0,0 +1,28 @@
+# Configuration for Cog ⚙️
+# Reference: https://github.com/replicate/cog/blob/main/docs/yaml.md
+
+build:
+  # set to true if your model requires a GPU
+  gpu: false
+
+  # a list of ubuntu apt packages to install
+  system_packages:
+    - "libgl1-mesa-glx"
+    - "libglib2.0-0"
+
+  # python version in the form '3.8' or '3.8.12'
```

Comment:
Do patch versions work now? They didn't use to but perhaps @bfirsh added that.

Reply:
I think they always have worked? It just passes down to pyenv which supports either. The thing I added was just using Docker's `python` image, which also supports either.

Reply:
I just did a test `cog build` with `python_version: "3.8.11"` and it appears to work:

```
 => [internal] load metadata for docker.io/library/python:3.8.11                                                                        1.1s
 => [stage-0 1/8] FROM docker.io/library/python:3.8.11@sha256:a0a734233420b17d9ab37125afc9d8217b75db153d55854ac6683e639f00a8e8         13.4s
```